src/weapon.c

```diff
@@ -513,6 +513,14 @@ int otyp;
 			bonn = 1;
 			bond = 4;
 		}
+		else if (obj->oartifact == ART_XIUHCOATL && otyp == BULLWHIP)
+		{
+			// couatl bite 
+			ocn = 1;
+			ocd = 2;
+			bonn = 2;
+			bond = max(4 + 2 * dmod, 2);
```

Comment:
As this is meant to be equal to the bite of a Couatl, perhaps it should follow the same logic as the Golden Sword of Y'ha-Talla and not be affected by dmod. Thoughts?